collections/Fairfax/MS_Fairfax_3.xml

```diff
@@ -37,67 +48,380 @@
+                        collar of SS (Gower was presented with a collar of SS by Henry Bolingbroke
+                        in 1393) with a badge, probably a swan, hanging from it (see Figure 6). The
+                        collar and badge, perhaps added later, are part of the Lancastrian livery,
+                        as well as the livery colours of white and blue noted by Harris, "Ownership
+                        and Readership', 121. S. Drimmer, <title>The Art of Allusion</title>, discusses
+                        the collar of SS in relation to limhers' treatment of authorial identity
+                        with Amans on p: 95 and Figure 38 on p. 98, illustrating the detail of the
+                        collar in colour plate 11.</decoNote>
+                     <decoNote type="border">Fairfax 3 was used by Macaulay as his copy-text, and
+                        his division of the poem into paragraphs and chapters' corresponds exactly
+                        to the hierarchy of the decorated initials. Six-line decorated initials,
+                        combined with bar-borders, mark the beginnings of all books except the
+                        Prologue (four-line). The borders consist of a single decorated column on
+                        the left, alone (Prol.) or with lateral extensions the width of the page at
+                        top and bottom (II, III, VI, VII) or at the bottom only (V); or a single
+                        decorated column between the text-columns with lateral extensions to left
+                        and right at top and bottom (IV, VIII) or at the bottom only (D). The
+                        branches are variously decorated with little ball-ornaments, trefoils,
+                        petals, sprays and occasionally thistles.</decoNote>
+                     <decoNote type="decInit"> Four-line decorated initials in gold, blue and pink,
+                        with pen-flourishing, mark major text-divisions, especially those that
+                        follow Latin verses and are marked by Macaulay as 'chapters' with small
+                        Roman numerals (e.g. 1.575, 1235, 1343).</decoNote>
+                     <decoNote type="flourInit"> Three-line initials, with flourishing, mark the
+                        less important major text-divisions (Macaulay's line-spaces). One-line
+                        decorated initials mark minor text-divisions (corresponding to Macaulay's
+                        paragraphs), and are also used for the Latin verses. In the French and Latin
+                        poems, the same kind of hierarchy of initials is observed, as appropriate to
+                        the different structure of the poems, but with no borders. </decoNote>
                   </decoDesc>
+                  <additions><list>
+                        <item>Fol. 1r, top margin, 'Sr Thomas fayrfax of Denton Knighte | true owner
```

Comment:
Personally I find it more useful to refer to this information with the `<provenance>` items.